build.gradle

```
@@ -763,6 +776,7 @@ subprojects {
     testImplementation 'org.mockito:mockito-core'
     testImplementation 'org.junit.jupiter:junit-jupiter-api'
     testImplementation 'org.junit.jupiter:junit-jupiter-params'
+    testImplementation 'net.jqwik:jqwik'
```

We probably don't want to add jqwik as a dependency for all unit tests.  We could add it automatically for `propertyTestImplementation` though as we'd expect it to be used by all property tests.
```suggestion
    propertyTestImplementation 'net.jqwik:jqwik'
```

---

Good call. I will fix this.